news/2018-01-10-MWC7K-2018-registrations-open.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "osu!mania 7K World Cup 2018: Registrations now open!"
+date: 2018-1-10 01:00:00 +0000
+---
+ 
+It's an exciting new year, and with that, our first announcement - osu!mania 7K World Cup is back for its 4th installment! Introducing a new host alongside an improved set of rules, this year's tournament promises to be more competitive than ever before. Registrations are now open, so what are you waiting for?
+![](/wiki/shared/news/banners/MWC7K_2018_Banner.jpg)
+ 
+For more details, refer to the **[osu!mania 7K World Cup 2018 wiki article](/wiki/Tournaments/MWC/2018/7K/)**.
+You can discuss this event and follow the most important changes in the **[official discussion thread](https://osu.ppy.sh/community/forums/topics/687190)**.
+ 
+## Tournament Schedule
+ 
++ Registration Phase: 10th - 24th January 2018
++ Drawings: 3rd February 2018 (14:00 UTC+0)
++ Group Stage: 10th - 11th February 2018
++ Round of 16: 17th - 18th February 2018
++ Quarterfinals: 24th - 25th February 2018
++ Semifinals: 3rd - 4th March 2018
++ Finals: 10th - 11th March 2018
++ Grand Finals: 17th - 18th March 2018
+ 
+## Prizes
++ **1st:** Profile badge, "osu!mania Champion" user title for one year
++ **2nd:** Profile badge
++ **3rd:** Profile badge
+
+_The rest of the prices will be determinated during the registration phase_
+ 
+## How do I register?
+ 
++ **[Click here to sign-up!](https://osu.ppy.sh/community/tournaments/13)**
++ Do not create teams in advance! Teams will be formed by country’s team captains after the registration phase has concluded.
```

Comment:
Teams will be formed by **the** country’s team captain after the registration phase has concluded.
(captains -> captain)